src/components/ParentCalculator.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Calculator from './calculator';
+
+const ParenCalculator = () => (
+  <div>
+    <Calculator />
+  </div>
+);
+
+export default ParenCalculator;
```

Comment:
- [ ] You did a great job working on this milestone! 🚀 However, I think that this "Parent Calculator" component is not getting any meaningful uses in this context, instead, to fulfill the requirement:

![image](https://user-images.githubusercontent.com/77069011/228219475-c91951c3-1c9d-4c45-a67b-ad4a3f305acd.png)

I suggest you Identify useful child components such as "CalculatorScreen", "Keypad", etc, these components make more sense and will let you organize your code better for the next steps of this project! 🤓 🔧 

If you're curious, [here's a very interesting article](https://react.dev/learn/thinking-in-react) from React's official docs, I strongly suggest you read it!